modules/contacts/setup.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+require_once APP_PATH.'modules/core/functions.php';
```

Comment:
core not included but contacts included? Is it possible? Doesn't seem the right place to include this file.

Reply:
This is a result of
https://github.com/cypht-org/cypht/issues/977#issuecomment-2103331522

Reply:
ok. I have moved the import to framework.php